Add fluentd to stability checker (#2598)
kyma-project/kyma: tools/stability-checker/README.md

2. Provision volume with a testing script, which then will be used by the Stability Checker. You can use the `local/provision_volume.sh` script. 
The script copies all files placed in the `local/input` directory to PV.

> 3. Logs from stability checker by default are not sent to Google Cloud Storage bucket. 

Reviewer: ```suggestion
3. The Stability Checker does not send logs to the Google Cloud Storage bucket by default. 
```

> 3. Logs from stability checker by default are sent to Google Cloud Storage bucket, to make it works create secret

Reviewer 2: IMO by default it should be disabled, to keep backward compatibility - this will require changes in the prowjob.

> To enable sending logs to bucket create secret named `sa-stability-fluentd-storage-writer` with credential to google account and

Reviewer: ```suggestion
To enable log forwarding, create the Secret with credentials to the google account and name it `sa-stability-fluentd-storage-writer`. 
```

> set `logsPersistence.enabled` parameter to `true` in `deploy/chart/stability-checker/values.yaml`.

Reviewer: ```suggestion
Additionally, set the **logsPersistence.enabled** parameter in the `deploy/chart/stability-checker/values.yaml` file to `true`.
```
